Berlin/BerOrOct1270.xml

```diff
@@ -200,14 +200,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 </q>
 <q n="2" xml:lang="gez">ምስለ፡ ፊቃሩ፡</q>
 <q xml:lang="en">With his band (belt).</q>
-<note>The first legend is written (in red ink) in the upper part of the miniature, above the saint's figure); the second legend is written (yellow ink?) at the right side of the saint’s figure. The handwriting is most probably the same as the one of the main text.</note>
+<note>The first legend is written (in red ink) in the upper part of the miniature, above the saint's figure; the second legend is written (yellow ink?) at the right side of the saint’s figure. The handwriting is most probably the same as the one of the main text.</note>
 </decoNote>
 <decoNote type="miniature" xml:id="d3">
 <locus target="#27v"></locus>
 <desc><ref type="authFile" corresp="AT1031VirginAndChild"></ref></desc>
-<q n="1" xml:lang="gez">እለ፡ ቅ<gap reason="illegible"></gap></q>
-<q n="2" xml:lang="gez">እግዝእትነ፡ ማርያም፡ <gap reason="illegible"></gap></q>
-<note>The first legend, almost unreadable, is written (in black ink) at the left side, upper corner; the second, also incomplete, because mostly unreadable, is written (black ink) above (at the right side) the Mary's figure. Other legends (portions of legends) are completely unreadable.  The handwriting looks recent, different from the one of the main text.</note>
+<q n="1" xml:lang="gez">ስእለ፡ ቅ<gap reason="illegible"></gap>ስ፡  <gap reason="illegible"></gap>ብር<gap reason="illegible"></gap>፡</q>
+<q n="1" xml:lang="en">Image of saint Gabriel</q>
+<q n="2" xml:lang="gez">ሥዕለ፡ እግዝእትነ፡ ማርያም፡ <gap reason="illegible"></gap>ስሌ፡  ፍቁራ፡ ወ<gap reason="illegible"></gap></q>
+<q n="2" xml:lang="en">Image of our Lady Mary, with her belowed...</q>
+<q n="3" xml:lang="gez">ስእለ፡ ቅ<gap reason="illegible"></gap>  ሚካ<gap reason="illegible"></gap></q>
+<q n="3" xml:lang="en">Image of saint Michael</q>
```

Comment:
the double numbering might cause problems. if you want to establish relations I wonder if it might be better to work with `<foreign>` instead

`<q n="1" xml:lang="gez">ስእለ፡ ቅ<gap reason="illegible"></gap>ስ፡  <gap reason="illegible"></gap>ብር<gap reason="illegible"></gap>፡<foreign xml:lang="en>Image of Saint <persName ref="PRS4377Gabriel"Gabriel</persName></foreign></q>`

Reply:
(same could be done elsewhere for translation)

Reply:
ok, thanks. Let´s try.